src/ImageSharp/Quantizers/Wu/WuQuantizer.cs

```diff
@@ -118,13 +103,13 @@ public sealed class WuQuantizer<TColor> : IQuantizer<TColor>
         /// </summary>
         public WuQuantizer()
         {
-            this.vwt = LongPool.Rent(TableLength);
-            this.vmr = LongPool.Rent(TableLength);
-            this.vmg = LongPool.Rent(TableLength);
-            this.vmb = LongPool.Rent(TableLength);
-            this.vma = LongPool.Rent(TableLength);
-            this.m2 = FloatPool.Rent(TableLength);
-            this.tag = BytePool.Rent(TableLength);
+            this.vwt = WuArrayPool.LongPool.Rent(TableLength);
```

Comment:
Agreed. I'd really like to see a more centralized approach. Glad you're making progress!